DetectionFactory/ConvertDetection.cs

```diff
@@ -0,0 +1,34 @@
+using Business.Detection.Common.Models;
+using Data;
+using Data.Detection;
+using FluentAssertions;
+using NUnit.Framework;
+
+namespace FactoryDetectionTests
+{
+    public class ConvertDetection
+    {
+        [SetUp]
+        public void Setup()
+        {
+        }
+
+        [Test]
+        public void DetectionFactoryTest()
+        {
+            var actualDetection = new MDetection {
```

Comment:
Should we also add the actualDetection and testDetection on Setup method? Let`s have a talk over this.

Reply:
Done